docs/ios/README.md

```diff
@@ -47,104 +146,263 @@ PSPDFKit.setLicenseKey("YOUR KEY");
+cd PSPDFKit-Demo
+```
+
+2. Open `config.xml` and change the deployment target to iOS 12 or later:
+
+```diff
+...
+  <platform name="android">
++   <preference name="AndroidXEnabled" value="true" />
+    <allow-intent href="market:*" />
+  </platform>
+  <platform name="ios">
+    <allow-intent href="itms:*" />
+    <allow-intent href="itms-apps:*" />
++   <preference name="deployment-target" value="12.0" />
+    ...
+  </platform>
+...
+```
+
+3. Add the PSPDFKit plugin:
+
+```bash
+cordova plugin add https://github.com/PSPDFKit/PSPDFKit-Cordova.git
+```
 
-1. Run `cordova create Cordova-Demo com.pspdfkit.demo CordovaDemo` to create a new Cordova project.
-2. Add a sample PDF into your `www` directory: `www/pdf/document.pdf`.
-3. Modify the `onDeviceReady` function in `www/js/index.js` like so:
+4. Add the iOS platform
+
+```bash
+cordova platform add ios
+```
+
+5. Add the PDF document you want to display in your project’s `www` directory. You can use <a href="https://pspdfkit.com/downloads/pspdfkit-ios-quickstart-guide.pdf" download="Document.pdf">this QuickStart Guide PDF</a> as an example.
+
+```bash
+cp ~/Downloads/Document.pdf www/Document.pdf
+```
+
+6. Open the www/js/index.js file:
+
+```bash
+open www/js/index.js
+```
+
+7. Modify the onDeviceReady function like so:
 
 ```javascript
 onDeviceReady: function() {
   this.receivedEvent('deviceready');
   // Set your license key here.
-  PSPDFKit.setLicenseKey("YOUR KEY");
-
-  // Show pdf with in single page mode.
-  PSPDFKit.present('pdf/document.pdf', {
-    pageMode: 'single',
-  });
+  PSPDFKit.setLicenseKey("LICENSE_KEY_GOES_HERE");
+  const DOCUMENT = (window.cordova.platformId === "ios") ? "Document.pdf" : "file:///android_asset/Document.pdf";
+  PSPDFKit.present(DOCUMENT);
 },
 ```
 
-4. `cd` into `Cordova-Demo` and run `cordova plugin add https://github.com/PSPDFKit/PSPDFKit-Cordova.git` to install the `pspdfkit-cordova` plugin.
-5. Open `config.xml` and change the deployment target to iOS 12 or later:
+8. The app is now ready to launch:
 
-```diff
-<platform name="ios">
-	<allow-intent href="itms:*" />
-	<allow-intent href="itms-apps:*" />
-+	<preference name="deployment-target" value="12.0" />
-</platform>
+```bash
+cordova emulate ios
+```
+
+### Running the PSPDFKit-Demo Cordova Example
+
+1. Open the Terminal app and clone the GitHub repository:
+
+```bash
+git clone https://github.com/PSPDFKit/pspdfkit-cordova.git
+```
+
+2. Change the current working directory to the example project’s folder:
+
+```bash
+cd pspdfkit-cordova/examples/cordova/PSPDFKit-Demo
 ```
 
-6. Run `cordova platform add ios` to add the iOS platform.
-7. If your application is targeting iOS versions **prior to iOS 12.2** and your application **does not already contain any Swift code**, then you need to make sure Xcode bundles Swift standard libraries with your application distribution. To to so, open your target Build Settings and enable `Always Embed Swift Standard Libraries`:
```

Comment:
Is this not needed anymore?

Reply:
No, because it's enabled by default for newly created projects with Xcode 12.4.